src/clj_http/multipart.clj

```diff
@@ -128,12 +128,14 @@
 (defn create-multipart-entity
   "Takes a multipart vector of maps and creates a MultipartEntity with each
   map added as a part, depending on the type of content."
-  [multipart {:keys [mime-subtype multipart-mode]
+  [multipart {:keys [mime-subtype multipart-mode multipart-charset]
```

Comment:
I'm not sure if the name `multipart-charset` is proper here. I'll appreciate any advices.

Reply:
I think that's a reasonable name, can you add some documentation for it?